groovin-music/src/components/Home/pages/HomeMain/HomeMain.css

```diff
@@ -0,0 +1,145 @@
+/*
+RecentlyPlayed Component
+Playlist Component
+*/
+.homepage-item-container {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0 auto;
+    justify-items: center;
+}
+
+.homepage-item {
+    width: 23%;
+    margin: 4px auto;
+    text-align: center;
+    border-radius: 5px;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    -ms-border-radius: 5px;
+    -o-border-radius: 5px;
+}
+
+@media (max-width: 800px) {
```

Comment:
I'm glad you're comfortable with CSS.  I'm decent but overall dislike it.  nice job!